skimage/filters/thresholding.py

```diff
@@ -384,3 +385,137 @@ def threshold_li(image):
             new_thresh = temp + tolerance
 
     return threshold + immin
+
+
+def threshold_multiotsu(image, nclass=3, nbins=256):
+    """Generates multiple thresholds for an input image. Based on the
+    Multi-Otsu implementation by Liao and Chung.
+
+    Parameters
+    ----------
+    image : array
+        Grayscale input image.
+    nclass : int, optional
+        Number of classes to be thresholded, i.e. the number of resulting
+        regions. Accepts an integer from 2 to 5.
```

Comment:
Can we check that nclass is in [2,5] in the code? And raise ValueError if not correct.